sensor_msgs/Image current_image

---
sensor_msgs/Image output_image
bool success